README.md

```diff
@@ -74,10 +115,74 @@ namespace:
 - **component parameters** 
   - are applied to a component's params.env file. There must be an entry whose key matches the component parameter. The params.env file is used to generate a ConfigMap. Entries in params.env are resolved as kustomize vars or referenced in a deployment or statefulset env section in which case no var definition is needed.
 
+Multiple overlays -
 
-Generating yaml output for any target can be done in the following way:
+Kfctl has the capability to combine more than one overlay during `kfctl generate ...`. An example is shown below where the profiles target in [manifests](https://github.com/kkasravi/manifests/tree/multiple_overlays/profiles) can include either debug changes in the Deployment or Device information in the Namespace (the devices overlay is not fully integrated with the Profile-controller at this point in time and is intended as an example) or **both**. 
```

Comment:
please link manifests to https://github.com/kubeflow/manifests/tree/master/profiles